src/App.js

```diff
@@ -19,22 +19,25 @@ const App = () => {
   const changeLanguage = (lng) => {
     i18n.changeLanguage(lng);
   };
-  
+
+  let currentLanguage = i18n.language;
+
   return (
-    <Router>
-        <LanguageButtons changeLanguage={changeLanguage} />
-        <ProjectHeader />
+    <Router basename={currentLanguage}>
```

Comment:
I didn't know about this, but now I do. :)

Reply:
I ended up not using basename anymore and used a wildcard in my path instead. But I am interested to see if anyone uses basename!